docs/quickstart.rst

```diff
@@ -18,3 +18,5 @@ provides significantly more detailed information.
    </building-apps/quickstart>`
 #. :doc:`Quick start: Launching MPI applications
    </launching-apps/quickstart>`
+#. :doc:`Quick start: Customizing run-time options
+   <mca/quickstart>`
```

Comment:
The other items in this list start with `/`.  Is the prefix `/` necessary?  (I honestly don't know).  I don't think that `/` assumes that the docroot is the top of the Open MPI docs, because (for example) https://docs.open-mpi.org/en/v5.0.x/ is the URL for the top of the v5.0.x docs, and that's clearly not at the docroot -- and the quickstart links work fine there.

Reply:
I believe it works since I tested this locally, but I can add the / for consistency